lib/matplotlib/cbook/__init__.py

```diff
@@ -286,8 +289,29 @@ class CallbackRegistry(object):
     functions).  This technique was shared by Peter Parente on his
     `"Mindtrove" blog
     <http://mindtrove.info/python-weak-references/>`_.
+
+
+    Parameters
+    ----------
+    exception_handler : callable, optional
+       If provided must have signature ::
+
+          def handler(exception: Exception) -> None:
```

Comment:
`def handler(exc: Exception) -> None` (for consistency with, e.g. the default handler below)